_data/projects/neo-mc.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: neo-mc
+desc: Next generation fork of Midnight Commander
+site: https://github.com/neo-mc/neo-mc
+tags:
+- c
+- glib
+- slang
+- cli
+upforgrabs:
+  name: wish-list
+  link: https://github.com/neo-mc/neo-mc/wiki/MCEditWishList
```

Comment:
I think this is the first time in my many years of reviewing project submissions that a wiki page has been used. I think this severely breaks the automation I've used for GitHub projects so it's gonna need some work to resolve this and get the PR passing.

If you're set on doing it this way, rather than using issues in the repository, please let us know.

Reply:
@psprint ^^